samples/sample-rdk-rules/requirements.txt

```
@@ -0,0 +1,3 @@
+s3==3.0.0
+boto3==1.20.32
```

Just bumbing the version here :)

```suggestion
boto3==1.24.51
```

---

Fixed